qt/applications/workbench/workbench/plotting/plotscriptgenerator/test/test_plotscriptgenerator.py

```diff
@@ -66,24 +64,42 @@
                           "ADS.retrieve(...)\n"
                           "\n"
                           "fig, axes = plt.subplots(...)\n"
-                          "axes[0].plot(...)\n"
```

Comment:
Can I ask if we are now missing tests for generating subplots with many Axes?

Reply:
I added one that explicitly tests that, it's in this file. I didn't like how it implicitly tested tiled plots instead of explicitly so I added the new test.